chainer/functions/__init__.py

```diff
@@ -180,6 +180,8 @@
 from chainer.functions.math.det import batch_det  # NOQA
 from chainer.functions.math.det import BatchDet  # NOQA
 from chainer.functions.math.det import det  # NOQA
+from chainer.functions.math.erf import erf  # NOQA
+from chainer.functions.math.erf import Erf  # NOQA
```

Comment:
We have decided not to expose newly added `Function`/`FunctionNode` classes under `chainer.functions`, so remove this line.